data/examples/shaders/source/shadertoy-MlKSWm.effect

```diff
@@ -0,0 +1,362 @@
+// https://www.shadertoy.com/view/MlKSWm v1.0 Converted by Surn Ubuntu Compatible
+
+#define vec2 float2
+#define vec3 float3
+#define vec4 float4
+#define ivec2 int2
+#define ivec3 int3
+#define ivec4 int4
+#define mat2 float2x2
+#define mat3 float3x3
+#define mat4 float4x4
+#define fract frac
+#define mix lerp
+#define iTime Time.x
+#define iResolution ViewSize
+#define PI 3.1415926535897932384626433832795
+
+uniform float2 mouse <
+	string name = "Virtual Mouse Coordinates";
+	string field_type = "slider";
+	float2 minimum = {0, 0};
+	float2 maximum = {100., 100.};
+	float2 scale = {.01, .01};
+	float2 step = {.01, .01};
+> = { 0., 0. };
+
+uniform float4x4 ViewProj <
+	bool automatic = true;
+>;
+
+uniform float4 Time <
+	bool automatic = true;
+>;
+
+uniform float4 ViewSize <
+	bool automatic = true;
+>;
+
+uniform float4x4 Random <
+	bool visible = false;
+	bool automatic = true;
+	string name = "Random Array";
+	string description = "A float4x4 value containing random values between 0 and 1";
+>;
+
+int2 iMouse()
+{
+	return int2(mouse.x * ViewSize.x, mouse.y * ViewSize.y);
+}
+
+float mod(float x, float y)
+{
+	return x - y * floor(x / y);
+}
+
+float2 mod2(float2 x, float2 y)
+{
+	return x - y * floor(x / y);
+}
+
+/*ps start*/
+uniform bool Burner <	
+	bool automatic = false;
+	bool visible = true;
+	string name = "Full Width";
+> = false;
+
+uniform float clip <
+	string name = "Flame Size";
+	string field_type = "slider";
+	float minimum = 0.0;
+	float maximum = 400.0;
+	float scale = 0.1;
+	float step = 0.25;
+	bool visible = true;
+> = 110.0;
+
+uniform float sparkGridHeight <
+	string name = "Spark Grid Height";
+	string field_type = "slider";
+	float minimum = 0.0;
+	float maximum = 200.0;
+	float scale = 1.0;
+	float step = 1.0;
+	bool visible = true;
+> = 50.0;
+
+uniform float flameMod <
+	string name = "Flame Modifier";
+	string field_type = "slider";
+	float minimum = -1.0;
+	float maximum = 2.0;
+	float scale = 1.0;
+	float step = 0.05;
+	bool visible = true;
+> = 1.0;
+
+uniform float flameTongueSize <
+	string name = "Flame Tongue Size";
+	string field_type = "slider";
+	float minimum = -.20;
+	float maximum = 0.30;
+	float scale = 1.0;
+	float step = 0.01;
+	bool visible = true;
+> = 0.02;
```

Comment:
Default value should match original shader. Due to the inversion, the correct default value is -0.01.